### ISSUE TITLE
Transition direction for pages ( rtl - ltr )

### ISSUE THREAD

**Issue by Yasser85** [woltapp/wolt_modal_sheet#266]
## Ability to choose rtl transition direction when navigate to another page

Hello
I'm working on an app that supports both Right-to-Left and Left-to-Right directions for multi languages by simply wrapping with Directionality widget.
I have a WoltModalSheet with 3 wolt pages in the HomePage.
The sliding animation direction between SliverWoltModalSheetPage is always goes LTR even when I set the direction to RTL in parent widget.
for example:
```dart
MaterialApp(
          builder: (context, child) {
            return Directionality(
              textDirection: TextDirection.rtl,
              child: child!,
            );
          },
          home: const HomePage(),
),
```
I'm not sure if there is a way or some work around to do that, please tell me if there is any.
But it will be nice if there was a direct way to handle the slide direction automatically or manually.
Thanks in advance.

**Comment by ulusoyca** [comment]
@MbIXjkee ^ Can we add this to our list when we refactor the page?

**Comment by ulusoyca** [comment]
@Yasser85 Transition direction is currently designed to resemble reading a book. When you open the next page, the page comes from right to left regardless of text direction. When you go to previous page, it comes from left to right similarly. Hence, we will probably won't change the default behavior, but we are working on more customization of the the animations.

**Comment by Yasser85** [comment]
> @Yasser85 Transition direction is currently designed to resemble reading a book. When you open the next page, the page comes from right to left regardless of text direction. When you go to previous page, it comes from left to right similarly. Hence, we will probably won't change the default behavior, but we are working on more customization of the the animations.

Actually, this is wrong. In all RTL languages (Arabic, Urdu, Farsi, Hebrew, .... ), the books go right to left. Which means when you open the next page, the page comes from left to right.
Also when you use something like page indicator or toggle buttons inside woltsheets to make it easy to navigate between sheets, then changing the active button or current indicator dot will go with the direction (which is RTL in this case), but the animation will go opposite which looks unsurprisingly weird. (I hope the idea is clear)

So, at least if there is a way to disable the transition animation completely, it will be appreciated.

Thanks for the good work and this nice package.

**Comment by ulusoyca** [comment]
Thanks! Today I learned something new. On Monday I will discuss with our motions designers 🫡 I will fix this accordingly.

**Comment by ulusoyca** [comment]
After discussing with the motion team internally, we realized that this is a bug we should fix.

**Comment by Yasser85** [comment]
OK, thank you.

**Comment by ulusoyca** [comment]
This is available now as part of 0.8.0
